lib/content-services/content-node-selector/content-node-selector-panel.component.spec.ts

```diff
@@ -462,6 +462,26 @@ describe('ContentNodeSelectorComponent', () => {
                 expect(component.showingSearchResults).toBeFalsy();
             });
 
+            it('should show nodes from the same folder as selected in the dropdown on clearing the search input', (done) => {
+                typeToSearchBox('piccolo');
+
+                setTimeout(() => {
```

Comment:
Using setTimeout in tests are considered a bad practice. async of fakeAsync should be used instead.
Since this test file contains tons of tests with setTimeout, after discussing it with Eugenio, we created an issue to clean up this test and remove the setTimeouts.